README.md

```diff
@@ -6,7 +7,13 @@ Cloudflow allows you to easily break down your streaming application to smaller
 Cloudflow integrates with popular streaming engines like Akka, Spark and Flink. It also comes with a powerful CLI tool to easily manage, scale and configure your streaming applications at runtime.
 With its powerful abstractions, Cloudflow allows to define, build and deploy the most complex streaming applications.
 
-In a nutshell, Cloudflow is an application development toolkit composed of:
+- Develop: Focus only on business logic, leave the boilerplate to us.
```

Comment:
```suggestion
- Develop: Focus only on business logic.
```

Reply:
@olofwalker I actually like the original statement. It describes part of what we do as we communicated here: https://www.lightbend.com/blog/cloudflow-released-lightbends-newest-open-source-project